packages/replay/src/canvas.ts

```diff
@@ -0,0 +1,52 @@
+import { getCanvasManager } from '@sentry-internal/rrweb';
```

Comment:
Yes, exactly - basically wrapping this, but providing a nicer way to add this than to ask people to pass some method into the replay config etc.

Also, it helps with CDN bundles, where this otherwise also very hard/impossible to solve 😬

Reply:
_should_ not be the case, but something is not quite right yet here 😅 so still need to look into it a bit more!